packages/bruno-electron/src/ipc/network/index.js

```diff
@@ -1047,6 +1047,10 @@ const registerNetworkIpc = (mainWindow) => {
                 scriptingConfig
               );
 
+              if (testResults?.nextRequestName !== undefined) {
```

Comment:
This also needs to be added to the CLI code: 

https://github.com/usebruno/bruno/blob/0988f2b86eae44db4af41169ce19c08339ac0161/packages/bruno-cli/src/runner/run-single-request.js#L342

Reply:
I have added the same to the CLI as requested but I am not sure how I can test that to make sure it is working as expected.

Reply:
@Its-treason and @martinsefcik can you please take a look again? I have done the suggested changes.